composer.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "wearejh/m2-module-adyen-payment",
   "description": "Adyen Payment module extension to support headless frontend",
-  "version": "3.0.2",
+  "version": "4.0.0",
```

Comment:
Please remove that completely. We should be controlling package version through tags in repository

Reply:
@bartoszkubicki I've removed it now